src/OrchardCore/OrchardCore.Localization.Core/PortableObject/PortableObjectStringLocalizer.cs

```diff
@@ -208,6 +209,11 @@ string ExtractTranslation()
                             // Extract translation without context
                             key = CultureDictionaryRecord.GetKey(name, null);
                             translation = dictionary[key, count];
+
+                            if (translation == null)
+                            {
+                                translation = dictionary.GetTranslationWithIgnoreContext(key);
+                            }
```

Comment:
Would need more time to understand what happens here. Not sure of the name, maybe `GetTranslationWithoutContext()`.

Did you tried it? Seems to do quite the same as `dictionary[key]` => `dictionary[key, null]`, the only difference i see is that you don't take into account the current `count` and you lookup in all `dictionary.Translations`, `Translations` being a `Dictionary<string, string[]>`. Does it mean that here we can't have a valid `count`?

If so maybe `GetTranslationWithoutCount()`, or maybe `GetTranslationWithoutPluralRule()`, not sure ...

Reply:
> Seems to do quite the same as dictionary[key] => dictionary[key, null]

Nope, coz If you look closely into the extension method, I'm looking for the key such as `RequiredAttributeDefaultMessage` in all the translations and the result something like `OrchardCore.DataAnnotations|RequiredAttributeDefaultMessage`

So `dictionary[key, null]` will look for key named `RequiredAttributeDefaultMessage` and not `OrchardCore.DataAnnotations|RequiredAttributeDefaultMessage`

Reply:
Okay, would need to focus on it more, but i trust you ;)

Sorry, i'm a little lost here, what would be the value of `count` here?

How the `DataAnnotationsDefaultErrorMessages` that you defined in another PR will be used?

Reply:
`DataAnnotationsDefaultErrorMessages` this just for the `PoExtractor` to extract the localized strings nothing more ;)

Reply:
> Sorry, i'm a little lost here, what would be the value of count here?

The `count` is omitted